File: TP3_Python/Codingan/KEYBOARD.py
from External import External

class KEYBOARD(External):
    def __init__(self, tipe_keyboard="", warna="", koneksi="", merk_external=""):
        super().__init__(koneksi, merk_external)
        self.tipe_keyboard = tipe_keyboard
        self.warna = warna

    def set_tipe_keyboard(self, tipe_keyboard):
        self.tipe_keyboard = tipe_keyboard

    def set_warna(self, warna):
        self.warna = warna

    def get_tipe_keyboard(self):
        return self.tipe_keyboard

    def get_warna(self):
        return self.warna

File: TP3_CPP/Codingan/KEYBOARD.cpp
#pragma once
#include "External.cpp"
#include <iostream>
#include <string>

using namespace std;

class KEYBOARD : public External
{
private:
    string TipeKeyboard;
    string Warna;

public:
    KEYBOARD()
    {
    }

    KEYBOARD(string TipeKeyboard, string Warna, string Koneksi, string MerkExternal) : External(Koneksi, MerkExternal)
    {
        this->TipeKeyboard = TipeKeyboard;
        this->Warna = Warna;
    }

    // Setter
    void setTipeKeyboard(string TipeKeyboard)
    {
        this->TipeKeyboard = TipeKeyboard;
    }

    void setWarna(string Warna)
    {
        this->Warna = Warna;
    }

    // Getter
    string getTipeKeyboard()
    {
        return TipeKeyboard;
    }

    string getWarna()
    {
        return Warna;
    }

    ~KEYBOARD()
    {
    }
};
